### ISSUE TITLE
State and Country field labels are not read by screen readers

### ISSUE THREAD

**Issue by anant1811** [woocommerce/woocommerce#27317]
**Prerequisites (mark completed items with an [x]):**
- [x] I have have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.
#17876 seems similar, I have left a comment there but wasn't sure if that should be re-opened.

**Describe the bug**
Country and State field labels are not read by screen readers. Please  see the below screenshots:

Correct behaviour:
![https://d.pr/i/aT2Fpc](https://d.pr/i/aT2Fpc+) 
Full Size: https://d.pr/i/aT2Fpc

Buggy behaviour:
![https://d.pr/i/vQ7zxJ](https://d.pr/i/vQ7zxJ+) 
Full Size: https://d.pr/i/vQ7zxJ

**Expected behavior**
State and country field titles should be readable by screen readers. 

**Actual behavior**

Screen readers skip reading the titles of state and country fields. 
**Steps to reproduce the bug (We need to be able to reproduce the bug in order to fix it.)**
Steps to reproduce the bug:
1. Add products to cart, go to checkout. 
2. Enable screen reader in operating system (on Mac, it's called "Voiceover" and is located in accessibility settings). Or you can use an actual screen reader if you have.
3. Cycle through the fields on checkout page. 

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
We use the [WooCommerce System Status Report](https://docs.woocommerce.com/document/understanding-the-woocommerce-system-status-report/) to help us evaluate the issue. 
Without this report we won't be able to fully evaluate this issue.
<details>
```
`
### WordPress Environment ###

WordPress address (URL): http://usecase.local
Site address (URL): http://usecase.local
WC Version: 4.3.1
REST API Version: ✔ 1.0.10
WC Blocks Version: ✔ 2.7.2
Action Scheduler Version: ✔ 3.1.6
WC Admin Version: ✔ 1.3.1
Log Directory Writable: ✔
WP Version: ❌ 5.4.2 - There is a newer version of WordPress available (5.5)
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_GB
External object cache: –

### Server Environment ###

Server Info: nginx/1.14.0
PHP Version: 7.2.9
PHP Post Max Size: 1,000 MB
PHP Time Limit: 1200
PHP Max Input Vars: 4000
cURL Version: 7.52.1
OpenSSL/1.0.2l

SUHOSIN Installed: –
MySQL Version: 5.7.23
Max Upload Size: 300 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 4.3.1
WC Database Prefix: wp_
Total Database Size: 3.75MB
Database Data Size: 2.14MB
Database Index Size: 1.61MB
wp_woocommerce_sessions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_actions: Data: 0.02MB + Index: 0.11MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_ms_snippets: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_options: Data: 1.03MB + Index: 0.06MB + Engine InnoDB
wp_postmeta: Data: 0.08MB + Index: 0.09MB + Engine InnoDB
wp_posts: Data: 0.05MB + Index: 0.06MB + Engine InnoDB
wp_snippets: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_term_relationships: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_usermeta: Data: 0.06MB + Index: 0.03MB + Engine InnoDB
wp_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_bookings_availability: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_bookings_availabilitymeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_booking_relationships: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_bundled_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_bundled_items: Data: 0.02MB + Index: 0.03MB + Engine InnoDB

### Post Type Counts ###

attachment: 2
bookable_resource: 1
page: 6
post: 1
product: 39
revision: 1
shop_order: 2

### Security ###

Secure connection (HTTPS): ❌
					Your store is not using HTTPS. Learn more about HTTPS and SSL Certificates.
Hide errors from visitors: ✔

### Active Plugins (4) ###

Bookings Helper: by WooCommerce – 1.0.3 – Installed version not tested with active version of WooCommerce 4.3.1
Woo Additional Tax Options: by wpnomad – 1.0.0 – Installed version not tested with active version of WooCommerce 4.3.1
WooCommerce Bookings: by WooCommerce – 1.15.23 – Installed version not tested with active version of WooCommerce 4.3.1
WooCommerce: by Automattic – 4.3.1 (update to version 4.3.2 is available)

### Inactive Plugins (9) ###

6.20 - More Setting Fields: by Zac Gordon – 1.0.0
Additional Tax Options: by Ashish Anant – 1.0 – Installed version not tested with active version of WooCommerce 4.3.1
AutomateWoo: by WooCommerce – 5.0.0
Code Snippets: by Code Snippets Pro – 2.14.0
WC Minimum Order Amount: by Hannah Swain – 1.0 – Installed version not tested with active version of WooCommerce 4.3.1
WooCommerce Product Add-ons: by WooCommerce – 3.0.36 – Installed version not tested with active version of WooCommerce 4.3.1
WooCommerce Product Bundles: by SomewhereWarm – 6.3.2
WooCommerce Subscriptions: by WooCommerce – 3.0.5 – Installed version not tested with active version of WooCommerce 4.3.1
WooCommerce Subscriptions - Custom Price String: by Prospress Inc. – 1.0.1 – Installed version not tested with active version of WooCommerce 4.3.1

### Settings ###

API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: booking (booking)
bundle (bundle)
external (external)
grouped (grouped)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –

### WC Pages ###

Shop base: #14 - /shop/
Basket: #15 - /cart/
Checkout: #16 - /checkout/
My account: #17 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Twenty Twenty
Version: 1.2 (update to version 1.5 is available)
Author URL: https://en-gb.wordpress.org/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build
personally we recommend using a child theme. See: How to create a child theme

WooCommerce Support: ✔

### Templates ###

Overrides: –

### Action Scheduler ###

Canceled: 1
Oldest: 2020-07-22 11:28:59 +0000
Newest: 2020-07-22 11:28:59 +0000

Complete: 38
Oldest: 2020-07-22 11:28:59 +0000
Newest: 2020-08-06 16:18:34 +0000

`
```
</details>

**Comment by juliaamosova** [comment]
Hi @anant1811,

Thank you for taking the time to report this bug, we really appreciate your help. I can reproduce it on my end. 

I made a screen recording to document how the screen reader reads the State field (it doesn't specify that the field that is being clicked on is the `State` field indeed):

https://cloudup.com/c117YXsPHla

Since this is related to accessibility, we should give it a high priority to fix.

**Comment by LisaLisa480** [comment]
Hello. Is there a status update on the fix for this bug?
